### ISSUE TITLE
Conflict between HTML and CSS color scopes

### ISSUE THREAD

**Issue by SmitAkbari26** [wesbos/cobalt2-vscode#227]
![image](https://user-images.githubusercontent.com/125372245/231370001-4bf1a1fe-5d12-4ca2-aef8-1c9086bb2373.png)

In above image you define all scope for html but see this,

![image](https://user-images.githubusercontent.com/125372245/231370194-f2131f74-1dc3-4fe4-9358-e533a4b1e163.png)

in this color coming from this scope foreground | entity.name.tag{ "foreground": "#9EFFFF" }

but in code color come from this css scope as you define

![image](https://user-images.githubusercontent.com/125372245/231370727-f0982585-2320-4baf-b73d-5828ae5e0093.png)

and you define html scope with this yellow color 

![image](https://user-images.githubusercontent.com/125372245/231371253-a0141501-3abc-4bed-b743-75d823ef7931.png)

So, basically as you define all html scopes and color for that scopes is not significant at all because there is no color coming as you define for html scopes.
